mmdeploy/backend/sdk/__init__.py

```diff
@@ -18,12 +18,15 @@
 
 if lib_path:
     lib_dir = os.path.dirname(lib_path)
-    sys.path.insert(0, lib_dir)
-
-if importlib.util.find_spec(module_name) is not None:
-    from .wrapper import SDKWrapper
-    __all__ = ['SDKWrapper']
-    _is_available = True
+    sys.path.append(lib_dir)
```

Comment:
should we update paths line 13-14 so that lib can be found?

Reply:
If you want to use sdk for pre-built package, you should install `mmdeploy_python` from the sdk package.

Reply:
It is not cause by `mmdeploy_python`,`SDKWrapper` require PyTorch...

Reply:
I have PyTorch in my env and pytorch2onnx, onnx2tensorrt works fine, but running `test.py` with for sdk failed.

Reply:
Try the latest commit.
Will `python -m 'import mmdeploy_python'` work?

Reply:
now it worked.